### PR TITLE
[tools] Support custom source language in translate.py

### PR DESCRIPTION
**Pull request by biodranik**
- Also added Hindi
- strings.txt format is printed last as more relevant

Used it in #5796

### CODE REVIEW

File: tools/python/translate.py [hunk 1]
```diff
@@ -149,8 +153,13 @@ def usage():
...
   text_to_translate = ' '.join(sys.argv[1:])
 
-  translations = deepl_translate(text_to_translate)
-  google_translations = google_translate(text_to_translate)
+  source_language = 'en'
+  if len(text_to_translate) > 3 and text_to_translate[2] == ':':
+    source_language = text_to_translate[0:2]
+    text_to_translate = text_to_translate[3:].lstrip()
+
+  translations = deepl_translate(text_to_translate, source_language)
```
Comment (sciunto) [inline tools/python/translate.py:161]: What's the benefits of calling deepl if there is no option to change the engine in the signature?

**Review by pastk — APPROVED** [review]
Would be good to update samples in TRANSLATIONS.md accordingly.

File: tools/python/translate.py (hunk 1 above)
Comment (biodranik) [reply tools/python/translate.py:161]: DeepL translations usually have a better quality than Google's.

Comment (sciunto) [reply tools/python/translate.py:161]: but your code is not using it right?

File: tools/python/translate.py [hunk 2]
```diff
@@ -149,8 +153,13 @@ def usage():
...
+  google_translations = google_translate(text_to_translate, source_language)
   translations.update(google_translations)
```
Comment (sciunto) [inline tools/python/translate.py:163]: just here, you are using google's one.

Comment (biodranik) [reply tools/python/translate.py:163]: DeepL does not support all languages that are already present in OM. Google Translate is used as a fallback for those unsupported languages.

**Pull request merged**